.github/workflows/ci-e2e.yaml

```diff
@@ -68,6 +68,8 @@ jobs:
       - name: Deploy
         run: |
           make docker-build-gateway-controller kind-load-gateway-controller docker-build-policy-controller kind-load-policy-controller deploy-gateway-controller
+          kubectl --context kind-mgc-control-plane -n multicluster-gateway-controller-system get deployments/mgc-controller-manager -o yaml | yq .spec.template.spec.containers[0].image
```

Comment:
I didn't see this log in the e2e test run. I would have expected to see it after line 254 and before 256 below. Is that an indication something isn't running as expected?
![Screenshot 2023-11-15 at 16 36 38](https://github.com/Kuadrant/multicluster-gateway-controller/assets/6498727/2877d1de-03fb-47c4-ad20-053ead0da1d4)

Reply:
ah i think this would be expected. 
the config changes need to merge before being applied right?